Multisig/Logic/PrivateKeyController.swift

```diff
@@ -26,8 +28,41 @@ class PrivateKeyController {
         }
     }
 
+    static func importKey(session: Session, installedWallet: InstalledWallet?) -> Bool {
+        do {
+            try KeyInfo.import(session: session, installedWallet: installedWallet)
+
+            Tracker.shared.setNumKeysImported(KeyInfo.count)
+            NotificationCenter.default.post(name: .ownerKeyImported, object: nil)
```

Comment:
do we need different tracking for wallet connect keys specifically?

Reply:
implemented